### ISSUE TITLE
ReturnCheck: enforceReturnType == false is broken

### ISSUE THREAD

**Issue by Gama11** [adireddy/haxe-checkstyle#225]
``` haxe
class Test {
    function foo() {
        return;
    }
}
```

``` json
{
    "checks": [
        {
            "type": "Return",
            "props": {
                "allowEmptyReturn": false,
                "enforceReturnType": false
            }
        }
    ]
}
```

Should not produce any warnings, but outputs:

```
./source/Test.hx:2: characters 1-32 : Info: Return type not specified for method "foo"
```

**Comment by T1mL3arn** [comment]
Strange, but I get the same issue. VS Code 1.74.1, checkstyle 2.8.4

```json
"defaultSeverity": "INFO",
"extendsConfigPath": "",
"numberOfCheckerThreads": 5,
"baseDefines": [],
"defineCombinations": [],

"checks": [
  {
    "type": "Return",
    "props": {
      "allowEmptyReturn": true,
      "enforceReturnType": false,
      "enforceReturnTypeForAnonymous": false
    }
  },
]
```

nvm, I read the docs: `It is currently built with haxe-checkstyle v2.4.1 and tokentree v1.0.1`
